libcxx/test/std/ranges/range.adaptors/range.stride.view/end.pass.cpp

```diff
@@ -0,0 +1,100 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+
+// constexpr auto end() requires(!__simple_view<_View>)
+// constexpr auto end() const requires(range<const _View>)
+
+// Note: Checks here are augmented by checks in
+// iterator/ctor.copy.pass.cpp.
```

Comment:
IMO this is kind of implied by the fact that we test different aspects of the class in different files. I'm not certain the comment carries its weight, since it makes one wonder if there's something special with this iterator that we do differently from all other iterators. I don't think that's the case.